<launch>

<rosparam file="$(find m3ros_control)/config/hardware_controllers.yaml" command="load" />
<!-- js controllers -->
<group ns="meka_roscontrol" >
  <node pkg="controller_manager" type="spawner" output="screen" name="meka_js" args="joint_state_controller"/>
</group>

</launch>

